packages/radio-group/src/Radio.tsx

```diff
@@ -67,7 +68,7 @@ const labelStyle = css`
 // Note colors are not in our palette
 const inputColorSet = {
   [Mode.Light]: css`
-    &:checked + ${inputDisplayWrapper.selector} > ${styledDiv.selector} {
+    &:checked + * ${inputDisplayWrapper.selector} > ${styledDiv.selector} {
```

Comment:
Huge red flags go off whenever I see `*` selectors. They present both performance and maintainability problems, and we should avoid them at all costs. We should find a different way to do this, even if it's a bit more work on our end.

Reply:
I think a quick alternative would be to add some `>` nesting instead but it would be somewhat brittle

Reply:
I changed it to use "* >" which should be significantly more performant since it's gone from:
"given all elements matching `styledDiv.selector` find all ancestors that have a checked input sibling" to
"given all elements matching `styledDiv.selector` find parents that have a checked input sibling"

Reply:
I'd **vastly** prefer that we use selectors not utilizing `*` at all for the other reasons I mentioned as well, not just performance. Using a child instead of descendent selector is an improvement, but not the issue I had  given the selector.